src/cloudWatchLogs/commands/viewLogStream.ts

```diff
@@ -87,6 +94,17 @@ export class DefaultSelectLogStreamWizardContext implements SelectLogStreamWizar
                 totalSteps: this.totalSteps,
             },
         })
+
+        // Add item to top of quick pick which indicates user instead wants to do Search Log Group quickpick flow
+        const separator = picker.quickPickSeparator()
+        const searchLogGroupItems: vscode.QuickPickItem[] = separator
+            ? [
+                  { label: 'Search Log Group', detail: 'Search within all Log Streams of the current Log Group' },
+                  { label: 'Log Streams', ...separator },
```

Comment:
similar to how "Log Streams" shows in the right column, I think we should use "Actions" as the common title for action items.